src/selectedtests/helpers.py

```diff
@@ -19,13 +20,14 @@ def get_evg_api():
     return RetryingEvergreenApi.get_api(auth=EvgAuth(evg_user, evg_api_key))
 
 
-def get_mongo_wrapper() -> MongoWrapper:
+def get_mongo_wrapper(mongo_uri: Optional[str] = None) -> MongoWrapper:
```

Comment:
I switched this to an optional so the calling code can use click's env variables and pass that in, instead of looking up the env var here.

Reply:
Since this method is no longer called by `src/selectedtests/work_items/cli.py`, is this change still needed? I believe this method is now only called by `src/selectedtests/app/app.py` which does not pass in mongo_uri

Reply:
yes, fixed.